mmtk/src/api.rs

```diff
@@ -37,6 +42,9 @@ pub extern "C" fn release_buffer(ptr: *mut Address, length: usize, capacity: usi
 
 #[no_mangle]
 pub extern "C" fn openjdk_gc_init(calls: *const OpenJDK_Upcalls, heap_size: usize) {
+    if cfg!(feature = "marksweep") {
```

Comment:
As we now allow run-time plan, it is not necessary to pass in a plan feature for OpenJDK. This would be incorrect if we do not build with `marksweep`.